packages/medusa/src/migrations/1657267320181-extended_batch_job.ts

```diff
@@ -0,0 +1,36 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class extendedBatchJob1657267320181 implements MigrationInterface {
+    name = 'extendedBatchJob1657267320181'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "batch_job" DROP COLUMN "status"`);
```

Comment:
Suggestion :
Since the previous migration, depending on the user version, can have either created the old batch job table or the new, i would do something like this:

- find if batch job exists and the column status exists
```
SELECT status
FROM information_schema.columns 
WHERE table_name='batch_job' and column_name='status';
```

- if there is a result then alter the table as you do, otherwise do nothing
- In the down method, we can just drop the table as usual

wdyt?